siemens_data.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import sys
+import pandas as pd
+from datetime import datetime
+
+""" Class for reading and manipulating Siemens data from transformed CSVs. 
+	Contains API for accessing data, as well as function for reading the CSV's. 
+"""
+
+class SiemensData:
+
+	data = None
+	datetimes = []
+
+	# Open and store transformed CSV data as class pd object. 
+	def read_csv(self, filename): 
+		self.data = pd.read_csv(filename, dtype=object)
+
+	def find_data_for_room(self, room_name):
+		equipment_dictionary = {} #Key = equipment name, value = sorted list of tuples containing (timestamp, data)
+
+		if self.data is None:
+			raise Exception("You haven't read in a csv yet!")
+
+		for header in self.data:
+			if room_name in header:
+				equipment_data = [] # Initialize container for (timestamp, data) tuples
+
+				equipment_name = self.find_equipment_name(header) #Key for dictionary
+				if len(self.datetimes) < 1: 
+					raise Exception("You haven't parsed the timestamps yet!")
+
+				for i in range(len(self.datetimes)):
+					datetime = self.datetimes[i]
+					data = self.data.get_value(i, header)
+					equipment_data.append((datetime, data))
+
+				equipment_dictionary[equipment_name] =  equipment_data
+
+		return equipment_dictionary
+
+
+
+	def find_equipment_name(self, header):
+		parts_of_header = header.split(".")
```

Comment:
You said that some headers are split by "." and some are split by ";" (etc). Can we just split on either, or will some headers that are split by "." include ";" not as a divider?

Reply:
I believe that it's either or. Right now it's a simple case of just figuring out which it is (it's always going to be either a ":" or a ".", never both) but this might not be the case if it's again different for other data files. Since we only have the data for one building, I'm worried there might be other conventions.